todo/src/App.js

```diff
@@ -1,15 +1,31 @@
-import React from "react";
+import React, { useEffect } from "react";
 import AddTodoForm from "./AddTodoForm";
 import './App.css';
 import TodoList from './TodoList';
 
+const useSemiPersistentState = () => {
+  const [todoList, setTodoList] = React.useState(JSON.parse(localStorage.getItem('savedTodoList')));
```

Comment:
What do you think will happen here if `localStorage.getItem('savedTodoList')` doesn't have a todoList saved yet?
Basically, what will happen on the initial run of this app with no data.

Reply:
Hey Grace, do you have any questions here?